addons/l10n_it/models/account_move.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _extend_with_attachments(self, attachments, new=False):
+
+        return\
+            super(AccountMove, self.with_context(account_predictive_bills_disable_prediction=True))\
+            ._extend_with_attachments(attachments, new)
```

Comment:
```suggestion
        super_with_context = super().with_context(account_predictive_bills_disable_prediction=True) 
        return super_with_context._extend_with_attachments(attachments, new)
```
Please avoid line continuation, nobody in the accounting team likes it. ^^

Reply:
Another solution
```suggestion
        return super(AccountMove, self.with_context(
            account_predictive_bills_disable_prediction=True,
        ))._extend_with_attachments(attachments, new)
```